### DISCUSSION TITLE
Remote SSH extension for VS Code

### DISCUSSION THREAD

**Discussion by stefancristianco** [blinksh/blink#2043] [Support]
Is there any way to install this extension?
Normally I should be able to install it from the .vsix file which you download from github, but this option "Install from vsix" is missing in UI.

**Comment by davidsi** [comment]
Do you mean in the remote device or your iPad? I had the same issue installing on the iPad , blink says "extension not available for web version".  But on the remote device I just used VLC in to the device and installed it normally

**Reply by stefancristianco** [reply]
I have vscode installed on my Ipad using blink. Now I need to open a remote project using the ssh extension from vscode.

**Comment by carloscabanero** [comment]
I think the remote SSH extension has been replaced by the VS Code tunnels themselves. This would be a question on VS Code itself.
